### PR TITLE
Bipt 45 RFC - Redirection to RFC page

### PR DESCRIPTION
**Pull request by abdulmalik97**
### Related Ticket

[BIPT - 43](https://eyblockchain.atlassian.net/jira/software/projects/BIPT/boards/301?selectedIssue=BIPT-43)

#### Dependent PR's

PR - 21

### Description

Created functionality to redirect the page to RFC page once submitted. RFC page shows all RFC proposals now.

### QA Instructions

- [ ] Click new Proposal 
- [ ] Start filling up all the fields and click on Submit RFC

Screenshot cannot showcase the transition thus no screenshot

### Checklist

- [x] I have reviewed the code changes myself
- [x] My code meets all of the acceptance criteria of the ticket it closes.
- [x] I have removed unused code (e.g., `console.logs`, commented out blocks, etc.)
- [ ] I have made all necessary changes to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] Any dependent changes have been merged and published.
- [x] Check PR title name is correct.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+### Related Ticket
```
Comment (meaghanfitzgerald) [inline .github/pull_request_template.md:1]: I am confused why this is part of your pr if it is already a part of the codebase. Did you add this manually or rebase using `git pull --rebase origin main`?

File: lib/useProposalsByStatus.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { useQuery } from "@tanstack/react-query";
+import axios from "axios";
+import { Proposal, RFCStatus, ProposalStatus } from "@prisma/client";
+
+type StatusCriteria = {
+  status: ProposalStatus;
+  rfcStatus: RFCStatus;
+};
+
+export function useProposalsByStatus(statusObject: StatusCriteria) {
```
Comment (meaghanfitzgerald) [inline lib/useProposalsByStatus.ts:10]: This function looks great. Will be very useful for populating our list pages, and we can possibly deprecate the `useDrafts` function and replace with this. Nice work!

**Comment by meaghanfitzgerald** [comment]
LGTM awaiting response to first comment before approving

File: .github/pull_request_template.md (hunk 1 above)
Comment (abdulmalik97) [reply .github/pull_request_template.md:1]: Discussed in the call.

File: lib/useProposalsByStatus.ts (hunk 2 above)
Comment (abdulmalik97) [reply lib/useProposalsByStatus.ts:10]: Resolved in the discussion in the call.

**Review by meaghanfitzgerald — APPROVED** [review]
lgtm

**Review by KaceyWood8 — APPROVED** [review]
Looks good to me!

**Review by Lana-Butorovic — APPROVED** [review]

**Comment by aj-may** [comment]
#### 📝 Changed routes:


| [**/rfc**](https://eybc-m1uafx2gd-ajmay.vercel.app/rfc) |
|:-:|
|<a href="https://eybc-m1uafx2gd-ajmay.vercel.app/rfc">Screenshot of /rfc</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/tX_qrowL_ijIJlzhTXK9axnSZl2GnAl_JvLx-b1sJ7GSO5LQ6qDSlRxgUsKEHRVVa2TWFFwNUHSqHQEEDb53oA.png">(view full size)</a> |



And 1 other route:
- [**/api/proposalByStatus**](https://eybc-m1uafx2gd-ajmay.vercel.app/api/proposalByStatus)



Commit 4713476f055ad32a26767739bb18082acf1fed50 (https://eybc-m1uafx2gd-ajmay.vercel.app).

**Pull request merged**